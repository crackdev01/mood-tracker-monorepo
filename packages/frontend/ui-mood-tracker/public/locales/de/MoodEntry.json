{
  "header": "Stimmungseintrag",
  "description": "Hallo! Wie geht es dir heute?",
  "moodStatus": {
    "placeholder": "Bitte wählen Sie Ihre Stimmung",
    "relaxed": "Entspannt",
    "motivated": "Motiviert",
    "energetic": "Energisch",
    "curious": "Neugierig",
    "confident": "Zuversichtlich"
  },
  "intensity": {
    "placeholder": "Intensität auswählen"
  },
  "table": {
    "headers": {
      "enteredAt": "Eingetragen bei",
      "status": "Status",
      "intensity": "Intensität"
    }
  },
  "deleteEntry": {
    "header": "Möchten Sie den folgenden Eintrag wirklich löschen?"
  },
  "entries": "Stimmungseinträge",
  "buttons": {
    "add": "Eintrag hinzufügen",
    "edit": "Eintrag bearbeiten",
    "delete": "Eintrag löschen",
    "yes": "Ja",
    "no": "Nein"
  }
}
