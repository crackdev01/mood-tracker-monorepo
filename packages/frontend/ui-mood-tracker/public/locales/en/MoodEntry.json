{
  "header": "Mood Entry",
  "description": "Hello! How are you feeling today?",
  "moodStatus": {
    "placeholder": "Please select your mood",
    "relaxed": "Relaxed",
    "motivated": "Motivated",
    "energetic": "Energetic",
    "curious": "Curious",
    "confident": "Confident"
  },
  "intensity": {
    "placeholder": "Select intensity"
  },
  "entries": "Mood entries",
  "buttons": {
    "add": "Add Entry",
    "edit": "Edit Entry",
    "delete": "Delete Entry"
  }
}
