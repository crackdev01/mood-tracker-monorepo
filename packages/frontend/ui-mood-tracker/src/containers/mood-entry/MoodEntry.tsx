import React from 'react';
import { useTranslation } from 'react-i18next';
import { Header } from 'semantic-ui-react';

import AddEntryModal from '../../components/mood-entry/add-entry/AddEntryModal';
import MoodList from '../../components/mood-entry/mood-list/MoodList';

import './mood-entry.scss';

const MoodEntry = () => {
  const { t } = useTranslation(['MoodEntry']);

  return (
    <article className="mood-entry">
      <section className="mood-entry__header">
        <Header as="h2">{t('header')}</Header>

        <Header as="h3">{t('description')}</Header>
      </section>

      <AddEntryModal />

      <MoodList />
    </article>
  );
};

export default MoodEntry;
