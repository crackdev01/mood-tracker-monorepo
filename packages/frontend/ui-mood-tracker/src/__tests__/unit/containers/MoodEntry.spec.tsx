import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import MoodEntry from '../../../containers/mood-entry/MoodEntry';

Enzyme.configure({ adapter: new Adapter() });

describe('MoodEntry', () => {
  test('renders', () => {
    const w = shallow(<MoodEntry />);
    expect(w.debug()).toMatchSnapshot();
  });
});
