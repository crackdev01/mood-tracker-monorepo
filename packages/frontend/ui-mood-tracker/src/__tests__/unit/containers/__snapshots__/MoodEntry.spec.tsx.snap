// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoodEntry renders 1`] = `
"<article className=\\"mood-entry\\">
  <section className=\\"mood-entry__header\\">
    <Header as=\\"h2\\">
      header
    </Header>
    <Header as=\\"h3\\">
      description
    </Header>
  </section>
  <AddEntryModal />
  <MoodList />
</article>"
`;
